inc/update.class.php

```diff
@@ -437,7 +437,6 @@ public function doUpdates($current_version = null) {
          case "9.4.0-dev":
             include_once("{$updir}update_93_94.php");
             update93to94();
-            break;
 
          case "9.4.0":
```

Comment:
Should'nt we add `9.4` here ? Seems that users that already migrated has `9.4` instead of `9.4.0` as current version.

Reply:
`9.4` should not exists. I've just done a test from a 9.3.x fresh install and 9.4.0 fresh install, update is correct